crates/biome_cli/tests/snapshots/main_commands_migrate_eslint/migrate_eslintrcjson_exclude_inspired.snap

```diff
@@ -22,14 +22,24 @@ biome.json migrate ━━━━━━━━━━━━━━━━━━━━
   i Configuration file can be updated.
   
     1   │ - {}
-      1 │ + {·"linter":·{·"rules":·{·"recommended":·false·},·"includes":·["**"]·}·}
+      1 │ + {·"linter":·{·"rules":·{·"recommended":·false·}·}·}
       2 │ + 
   
 
 ```
 
 ```block
-Run the command with the option --include-inspired to also migrate inspired rules.
+migrate ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  i 100% (1/1) of the rules can be migrated.
+  
+  i Rules that can be migrated to an inspired rule using --include-inspired:
+  
+  - no-else-return
+  
+  i Rules that can be migrated to a nursery rule using --include-nursery:
+  
```

Comment:
Looks like this section should be skipped if there are no rules in that category.